lib/pbench/server/api/resources/query_apis/metadata_index/iteration_samples.py

```diff
@@ -223,23 +224,27 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
 
         Args:
             json_data:
-            "run_id": run id of a dataset that client wants to search for
-                    fetching the iteration samples.
+                "run_id": run id of a dataset that client wants to search for
+                        fetching the iteration samples.
 
-            "scroll_id": Server provided Elasticsearch scroll id that client
-                        recieved in the result of the original query with
-                        filters. This can be used to fetch the next page of
-                        the result.
+                "scroll_id": Server provided Elasticsearch scroll id that client
+                            recieved in the result of the original query with
+                            filters. This can be used to fetch the next page of
+                            the result.
 
-            "filters": key-value representation of query filter parameters to
-                       narrow the search results e.g. {"sample.name": "sample1"}
+                "filters": key-value representation of query filter parameters to
+                           narrow the search results e.g. {"sample.name": "sample1"}
 
-        EXAMPLE:
+        EXAMPLES:
             {
                 "run_id": "1234567"
-                "scroll_id": "cmFuZG9tX3Njcm9sbF9pZF9zdHJpbmdfMg=="
                 "filters": {"sample.name": "sample1"},
             }
+            or
+            {
+                "run_id": "1234567"
+                "scroll_id": "cmFuZG9tX3Njcm9sbF9pZF9zdHJpbmdfMg=="
+            }
```

Comment:
In the case where the request provides a `"scroll_id"`, our only use for `"run_id"` is to log the request.

I'm not saying that that is good or bad, and it doesn't seem onerous to require the requestor to include it, but, looking under the covers, it's slightly weird; however, if we were to change the underlying database in the future, we might need the run ID then, so perhaps it's good to require it even though we don't really need it.

Reply:
`run_id` is not only used to log the request, it as also used to check authorization against the ownership of the dataset, since we can not get user information from `scroll_id`

Reply:
Ah, OK.  Nevermind.  🙂